src/hume/empathic_voice/chat/audio/audio_utilities.py

```diff
@@ -0,0 +1,31 @@
+"""Utilities for audio playback."""
```

Comment:
Makes me a little nervous this is a public method in a public module.. what do you think?

Reply:
Updated this method and effectively everything except MicrophoneInterface (from `microphone_interface.py`) and Stream (from `asyncio_utilities.py`) to be private (via single underscores). MicrophoneInterface and Stream will be the only direct importable modules (i.e. `from hume import MicrophoneInterface`, `from hume import Stream`).
- See relevant `fern-config` branch & commit here: https://github.com/HumeAI/fern-config/commit/0569bd97a4734efee6e0e0944e45cd0ad6b33140